servatrice/src/server_logger.cpp

```diff
@@ -57,7 +57,7 @@ void ServerLogger::logMessage(QString message, void *caller)
     // filter out all log entries based on values in configuration file
     bool shouldWeWriteLog = settingsCache->value("server/writelog", 1).toBool();
     QString logFilters = settingsCache->value("server/logfilters").toString();
-    QStringList listlogFilters = logFilters.split(",", QString::SkipEmptyParts);
+    QStringList listlogFilters = logFilters.split(",", Qt::SkipEmptyParts);
```

Comment:
Qt::SkipEmptyParts was introduced in Qt 5.14, I'm afraid we'll need an #ifdef here

Reply:
I just noticed a lot more warnings as well :(